### ISSUE TITLE
[EN][Core/Dungeon] Random Game Crash during dungeon (= Dead outside instance and party bug)

### ISSUE THREAD

**Issue by BeVeR86** [Atlantiss/BugTracker#2439]
Hi,

Since the LFG update 2 days ago I have a strange problem. Sometimes when I am doing a dungeon my game randomly crashes (it seems to happen always in a boss fight though it might just be random). Anyway when I start my game again and log back into the server I am dead (like the Alt+F4 bug where you are dead when you alt+f4 while in combat) but not only that I am at the spirit healer as a ghost and I am no longer in the party.
However to the group I am still in the party (and dead) and my body is still in the instance.
If someone resses me or I walk back into the dungeon entrance I get teleported back to where I was before we started the dungeon.

The biggest problem is: the party can not kick me from the group (as I am not actually in the group it just shows it like that to them. Therefore they can't invite me back to the group either (as the group is already full).
So basically I get locked out of the dungeon for no reason (I'm not doing anything strange my game client just freezes and crashes, no error message either).

This is a major inconvenience and when it happens it pisses me off so bad, especially when it is near the last boss.

I really hope you guys can find where these problems come from. It seems to be a combination of multiple things going wrong.

If I find any extra info I will post it, but so far it seems pretty random to me.

**Comment by Aival94** [comment]
Try to do some rdfs with a new client to check if a general bug or just you specific client bug as i have not any suck bug after last update.

**Comment by BeVeR86** [comment]
The crash itself might be a client issue (though I know of another person who has experienced this problem too) but not being in the party while for the rest of the group you are in the party doesn't sound like something that would be client side. However I will try a different client, hopefully my crashes will go away. :)

**Comment by xolon** [comment]
everyone who uses atlantiss client also has a chance to randomly crash in bgs, ive noticed this few times when everyone around me just went offline ( i think this crashes don't affect me coz i use molten client with atlantiss launcher)

**Comment by cosminush** [comment]
I want to report and an bug about that cd on rdf, when kick an player or he get out by himself, that cd affect the remained players and cannt enter again in rdf for last man.
